metricbeat/helper/prometheus/prometheus.go

```diff
@@ -35,7 +35,11 @@ import (
 	"github.com/elastic/beats/v7/metricbeat/mb"
 )
 
-const acceptHeader = `application/openmetrics-text; version=0.0.1,text/plain;version=0.0.4;q=0.5,*/*;q=0.1`
+const (
+	promTextHeader    = `text/plain;version=0.0.4;q=0.5,*/*;q=0.1`
+	openmetricsHeader = `application/openmetrics-text; version=0.0.1,text/plain;version=0.0.4;q=0.5,*/*;q=0.1`
```

Comment:
@newly12 in order to have it in can we remove `openmetricsHeader` since it's not used at all? We can just update the initial `acceptHeader` removing the `application/openmetrics-text; version=0.0.1` part.

Reply:
done